xpdacq/analysis.py

```diff
@@ -154,16 +154,18 @@ def save_tiff(headers, dark_subtraction=True, *, max_count=None):
             try:
                 # bluesky only looks for uid it defines
                 dark_search = {'group': 'XPD',
-                               'sc_dark_uid': dark_uid_appended} # the one we need to look up data
+                               'uid': dark_uid_appended} # the one we need to look up for dark
```

Comment:
Can you just use `header.start['sc_dk_field_uid']` here, or do you need it again later?